File: backend/src/003-archivos/read_write/read_write.rb
#!/usr/bin/ruby

# Abrir un archivo para escritura
File.open("ejemplo.txt", "w") do |archivo|
    # Escribir en un archivo
    archivo.write("Contenido del archivo\n")
end

# Abrir un archivo para lectura
File.open("ejemplo.txt", "r") do |archivo|
    # Leer un archivo
    archivo.each_line do |linea|
        puts linea
    end
end

File: backend/src/003-archivos/read_write/read_write.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

# Abrir archivos en modo escritura
with open("ejemplo.txt", "w") as archivo:
    # Escribir en archivos
    archivo.write("Contenido del archivo\n")

# Abrir archivos en modo lectura
with open("ejemplo.txt", "r") as archivo:
    # Leer archivos
    contenido = archivo.read()
    # Obtener contenido en lista
    lineas = archivo.readlines()
